drsm/drsm.go

```diff
@@ -58,7 +58,8 @@ type Drsm struct {
 	ipModule            ipam.Ipamer
 	prefix              map[string]*ipam.Prefix
 	mongo               *MongoDBLibrary.MongoClient
-	globalChunkTblMutex sync.Mutex
+	globalChunkTblMutex sync.RWMutex
+	punchLivenessTime   int
```

Comment:
Commenting out this variable (which is unused) will fix the lint issue
```suggestion
	// punchLivenessTime   int
```